lib/DDG/Spice/IndegoBikeshareLocations.pm

```diff
@@ -0,0 +1,34 @@
+package DDG::Spice::IndegoBikeshareLocations;
```

Comment:
There are multiple bikeshare companies but usually one per city and each will have a different brand name (Indego) or sponsor name (Citi)